backend/src/main/java/se/chalmers/dat265/group1/model/SlmData.java

```diff
@@ -0,0 +1,28 @@
+package se.chalmers.dat265.group1.model;
+
+public class SlmData extends DataModel{
+
+    private int id;
+    private int physicalPrintID;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "physicalPrintID" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)